rlclientlib/azure_factories.cc

```diff
@@ -15,11 +17,15 @@ namespace reinforcement_learning {
   int observation_sender_create(i_sender** retval, const u::configuration&, error_callback_fn*, i_trace* trace_logger, api_status* status);
```

Comment:
Why is `observation_sender_create` a different operation to `observation_api_sender_create`? If the parameters are the same is the difference not in the concrete implementation of the sender?

Reply:
One uses event hub and another uses http apim auth and different clients and mapped to different types. Not sure what you are suggesting here. Sender works with both auth.

    sender_factory.register_type(value::OBSERVATION_EH_SENDER, observation_sender_create);
    sender_factory.register_type(value::INTERACTION_EH_SENDER, interaction_sender_create);
    sender_factory.register_type(value::OBSERVATION_HTTP_API_SENDER, observation_api_sender_create);
    sender_factory.register_type(value::INTERACTION_HTTP_API_SENDER, interaction_api_sender_create);

Reply:
Is it possible to use both at the same time? Or just one?